api.js

```diff
@@ -64,13 +57,11 @@ module.exports = function (plugins) {
     if(!plug.init)
       throw new Error('plugins *must* have "init" method')
 
-    if(isString(plug.name))
-      if(find(create.plugins, function (_plug) {
-        return _plug.name === plug.name
-      }))
+    var name = plug.name
+
+    if(isString(name) && create.plugins.indexOf(name) !== -1)
```

Comment:
this doesn't do the same thing. `[{name: 'foo'}].indexOf('foo') == -1` this will always return -1.